src/sgl/core/bc_codec.cpp

```
@@ -0,0 +1,1073 @@
+        m_impl = std::make_unique<BCCodecNVTTImpl>();
+    else
+        m_impl = std::make_unique<BCCodecSWImpl>();
+#else
+    (void)prefer_nvtt;
+    m_impl = std::make_unique<BCCodecSWImpl>();
+#endif
+}
+
+BCCodec::~BCCodec() = default;
+
+bool BCCodec::is_nvtt_available()
```

Cppcheck (2.21.0)

[style] The function 'is_nvtt_available' is never used.

(unusedFunction)